test/OpenFeature.Contrib.Providers.GOFeatureFlag.Test/HttpClientMock.cs

```diff
@@ -1,34 +0,0 @@
-using System.Net;
```

Comment:
@askpt was this file just unused?

cc @thomaspoignant

Reply:
Yes. This file was not used for tests. Likely some dead code from previous tests.